public/locales/en/modules/torrents-status.json

```diff
@@ -19,13 +19,19 @@
       "labelFilter": {
         "label": "Label list",
         "description": "When 'is whitelist' checked, this will act as a whitelist. If not checked, this is a blacklist. Will not do anything when empty"
+      },
+      "displayRatioWithFilter": {
+        "label": "Display filtered torrents list ratio",
+        "description": "If disable, only global ratio will be display. Global ratio still use labels if set"
```

Comment:
```suggestion
        "description": "If disabled, only the global ratio will be display. The global ratio will still use the labels if set"
```

Reply:
Totally agree : I try to make short sentence but even if I understand English, I don't really speak it